### ISSUE TITLE
AntiBarrel causing world crash with 362 items

### ISSUE THREAD

**Issue by PaulD61** [TheLimePixel/Modern-Skyblock-3#753]
The 'almost infinite' storage was causing the game to freeze with only 362 items in it. I'm storing all the enchanted diamond armor and tools from loot bags.
It was making the whole game freeze if I tried to add or remove an item using the gui. This meant that I had to cancel the game from the control panel and reload.

I've changed nothing but after about 5 freezes I'm now crashing the world when I select the AntiBarrel. The game itself remains in memory and I can successfully reload the world.
Gave this message;
**Internal Exception: java.lang.IllegalArgumentException: Payload may not be larger than 32767 bytes**

I've managed to hopper the items out and I can now open the AntiBarrel interface again.
I have four other AntiBarrels in use without problem and they seem to be working fine. This is the only one containing enchanted armor.

Game running in single player in 12GB RAM, not on a server.
No crash report produced

**Comment by PaulD61** [comment]
The same is now happening with a different barrel filled with bees (princesses).
I'm getting world crashes so thankfully I don't need to reload the pack.
Same message as above.
Using a hopper I pulled out 1 complete species (34 Modest Princesses) to stop the crashing so the barrel is now reporting 143 items, 30/100000 types. It didn't stop crashing until I'd completely removed the species so I guess the crash is tied to types rather than items.
Crashed again as soon as I inserted a single Modest bee so that confirms it, it's 'types'.
